public/data/more-info.json

```diff
@@ -0,0 +1,27 @@
+[{
+		"id": 1,
+		"name": "XYZ Homeless Shelter",
+		"description": "A description of the non-profit.",
+		"address": "12345 Road Drive",
+		"phone": 1234567890,
+		"website": "www.example.com",
+		"email": "hello@example.com",
+		"items": [{
+			"quality": ["quality check 1", "quality check 2"],
+			"instructions": ["instruction 1", "instruction 2"]
+		}]
```

Comment:
Could you elaborate more on the inner structure of `items`? What is the meaning of  `quality ` and `instructions `?

Reply:
Instead of using `quality check 1`, `quality check 2`, maybe switching to real-world example will also help.

Reply:
I think we will need a 'name' field for each item as well.